ios/FluentUI/Bottom Sheet/BottomSheetTokenSet.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public class BottomSheetTokenSet: ControlTokenSet<BottomSheetTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        case backgroundColor
+        case cornerRadius
+        case shadow
+    }
+
+    init() {
+        super.init { token, theme in
+            switch token {
+            case .backgroundColor:
+                return .dynamicColor { DynamicColor(light: GlobalTokens.neutralColors(.white),
```

Comment:
Would returning `theme.aliasTokens.backgroundColors[.neutral1]` do the trick instead?

Reply:
It would! But I just realized I had the wrong color token (supposed to be gray900, not black)